SeniorInterview/StocksAnalyzed/src/AggregateRequestor.java

```diff
@@ -0,0 +1,59 @@
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.ArrayList;
+
+public class AggregateRequestor extends Requestor {
+
+    public String request(String ticker, String startDate, String endDate){
+        String grouppedURL = String.format("https://api.polygon.io/v2/aggs/ticker/%s/range/1/day/%s/%s?adjusted=true&sort=asc&limit=120&apiKey=%s",
```

Comment:
I think I'd be interesting to have this URL as a const, making it easier to update whenever needed